HTML/White Arrows.html

```diff
@@ -0,0 +1,473 @@
+                        <th>▽</th>
+                        <th>◁</th>
+                        <th>▷</th>
+                    </tr>
+                    <tr>
+                        <th style="background-color: #00ffff">▽</th>
+                        <th>▽</th>
+                        <th>◁</th>
+                        <th>▷</th>
+                        <th>△</th>
+                        <th>◁</th>
+                        <th>△</th>
+                        <th>△</th>
+                    </tr>
+                    <tr>
+                        <th style="background-color: #00ffff">◁</th>
+                        <th>◁</th>
+                        <th>▷</th>
+                        <th>△</th>
+                        <th>◁</th>
+                        <th>△</th>
+                        <th>▷</th>
+                        <th>◁</th>
+                    </tr>
+                </table>
+                <table>
+                    <tr>
+                        <th style="background-color: black"><p style="color: #ffffff">Teal</p></th>
+                        <th>1st</th>
+                        <th>2nd</th>
+                        <th>3rd</th>
+                        <th>4th</th>
+                        <th>5th</th>
+                        <th>6th</th>
+                        <th>7th</th>
+                    </tr>
+                    <tr>
+                        <th style="background-color: #008080">△</th>
+                        <th>◁</th>
+                        <th>△</th>
+                        <th>▷</th>
+                        <th>△</th>
+                        <th>▷</th>
+                        <th>▷</th>
+                        <th>▽</th>
+                    </tr>
+                    <tr>
+                        <th style="background-color: #008080">▷</th>
+                        <th>▷</th>
+                        <th>▽</th>
+                        <th>△</th>
+                        <th>◁</th>
+                        <th>△</th>
+                        <th>▽</th>
+                        <th>△</th>
+                    </tr>
+                    <tr>
+                        <th style="background-color: #008080">▽</th>
+                        <th>▽</th>
+                        <th>▷</th>
+                        <th>◁</th>
+                        <th>▽</th>
+                        <th>◁</th>
+                        <th>△</th>
+                        <th>▷</th>
+                    </tr>
+                    <tr>
+                        <th style="background-color: #008080">◁</th>
+                        <th>△</th>
+                        <th>◁</th>
+                        <th>▽</th>
+                        <th>▷</th>
+                        <th>▽</th>
+                        <th>◁</th>
+                        <th>◁</th>
+                    </tr>
+                </table>                
+        </div>
+        <div class="page-footer relative-footer">Page 3 of 3</div>
+    </div>
```

Comment:
Spacing is off starting on the last page, I think you might also be missing a `<div>` in the end of the document. Line everything up and it'll be easy to see.

Reply:
Not sure if <div> with section needs closing tag because i looked in the other manual and it doesnt have one.

Reply:
All div ending tags are required according to w3. If you see them missing, that usually are mistakes which I have seen in some manuals in the past.

Reply:
Yeah in 0 manual div section is missing a closing tag